packages/nimble/R/nimbleFunction_core.R

```diff
@@ -117,6 +117,30 @@ nimbleFunction <- function(setup         = NULL,
     return(generatorFunction)
 }
 
```

Comment:
Thanks the wiki makes this look easy to use. Could you add a comment on the function that links to the wiki, so as to improve discoverability of documentation? More generally, what do you think of more generally adding documentation to internal functions?